screens/CameraRecording.tsx

```diff
@@ -140,10 +142,21 @@ const CameraRecording: FunctionComponent<CameraRecordingProps> = ({
         />
       )}
       {countdownTime >= 0 && !recording && (
-        <View style={Styles.overlayView}>
+        <View
+          style={{
+            ...Styles.overlayView,
+            backgroundColor: overlayBackgroundColor,
+          }}
+        >
           {overlay}
           <Text
-            style={Styles.beginText}
+            style={{
+              ...Styles.beginText,
+              color:
+                overlayBackgroundColor === "rgba(0, 70, 67, .93)"
```

Comment:
Throw this in as a constant?

Reply:
That would have been way too smart for me. Will do